providers/sendinblue/src/lib/sendinblue.provider.spec.ts

```diff
@@ -4,6 +4,8 @@ import { EmailEventStatusEnum } from '@novu/stateless';
 const mockConfig = {
   apiKey:
     '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',
+  from: 'test@novu.co',
```

Comment:
What's the reason that the test pass if we add this and we don't change the mocked message that has a different `from` email? Not sure if I am missing anything but it doesn't feel right. Or maybe we should review the validity of the tests.
Would you mind to review that a bit?

Reply:
The reason was because the tests were only checking if `sendMessage(mockNovuMessage)` was called with the right arguments. The current test were not checking if `provider.transactionalEmailsApi.sendTransacEmail` was called with the right defaults applied within `sendMessage`.